arch/arm/soc/st_stm32/stm32l4/Kconfig.defconfig.stm32l433xx

```diff
@@ -0,0 +1,27 @@
+# Kconfig - ST Microelectronics STM32L433XX MCU
+#
+# Copyright (c) 2016 Open-RnD Sp. z o.o.
```

Comment:
You introduce a new file. Please add your copyright in the header and remove the others.

Reply:
Sure, I can add my own, but I don't think I should remove the existing headers since I copied this from the L432 file(??) ... can you confirm I should remove the existing headers even if the base file had them?

Reply:
Yes. When you introduce a new file, you add your copyright notice.